Copy to SD Card root directory to update/config.ini

```diff
@@ -202,31 +218,51 @@ ack_notification:1
 notification_m117:0
 
 #### Emulate M600
-# The TFT intercepts the M600 gcode (filament change) and emulates the logic instead of demanding it to Marlin firmware.
+# The TFT intercepts the M600 gcode (filament change) and emulates the logic instead of
+# demanding it to Marlin firmware.
```

Comment:
"demanding it **_from_** Marlin" or maybe "sending it to Marlin"

Reply:
I think "to" should be more correct. The statement means that the M600 logic is handled by Marlin while "from" is something retrieved from Marlin and then used by TFT

Reply:
You cannot demand something TO something/somebody only FROM something/somebody.
demand ~= request in a mandatory way

Reply:
so something like "The TFT intercepts the M600 gcode (filament change) and emulates the handling logic otherwise provided by marlin firmware" should be correct

Reply:
Seems perfect!